src/components/subscriptions/edit/EditSubscription.js

```diff
@@ -192,31 +192,24 @@ function EditSubscriptionForm(props) {
                 validate={(value) => nonEmptyField(value, i18nUtil)}
             />
 
-            <Field name="plan_name">
-                {({ ...props }) => (
-                    <FastField
-                        component={FormTextField}
-                        variant="outlined"
-                        select
-                        size="small"
-                        {...props}
-                        {...selectProps}
+            <FastField
+                name="plan_name"
+                component={FormTextField}
+                variant="outlined"
+                select
+                size="small"
+                {...selectProps}
```

Comment:
I don't think `selectProps` exists in this context, so this line can be removed.

Reply:
I believe that is in use to declare id and label. I left that from the reference I used because I think it cleans up that component a bit.

Reply:
Oops, I must have overlooked that in the last PR, but I see it now 👍